lib/node_modules/@stdlib/stats/base/dists/weibull/cdf/benchmark/benchmark.js

```diff
@@ -32,17 +33,22 @@ var cdf = require( './../lib' );
 
 bench( pkg, function benchmark( b ) {
 	var lambda;
+	var len;
 	var k;
 	var x;
 	var y;
 	var i;
 
+	len = 100;
+	x = new Float64Array( len );
+	lambda = new Float64Array( len );
+	k = new Float64Array( len );
 	b.tic();
-	for ( i = 0; i < b.iterations; i++ ) {
-		x = ( randu()*100.0 ) + EPS;
-		lambda = ( randu()*100.0 ) + EPS;
-		k = ( randu()*100.0 ) + EPS;
-		y = cdf( x, k, lambda );
+	for ( i = 0; i < len; i++ ) {
```

Comment:
```suggestion
	for ( i = 0; i < b.iterations; i++ ) {
```

The range remains the same.